.travis.yml

```diff
@@ -7,11 +7,15 @@ sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bower's cache
 
 env:
+  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
   - EMBER_TRY_SCENARIO=default
```

Comment:
so I studied semver a little more and a version of `^2.6.0` include everything greater than a particular version in the same major range. So this resolves to `Ember 2.10.1`, which means we can remove the `default` scenario because it is the same as the `release` scenario.